tests/execute/result/basic.sh

```diff
@@ -85,7 +85,7 @@ EOF
     rlPhaseEnd
 
     rlPhaseStartTest "Verbose execute prints result - abort case"
-        rlRun -s "tmt run --id \${run} --scratch --until execute tests tests -n /abort provision --how container execute -v" "2"
+        rlRun -s "tmt run --id \${run} --scratch --until execute tests tests -n /abort provision --how container execute -v 2>&1 >/dev/null" "2"
```

Comment:
Regarding the output catching: Here, for example we're using `rlRun -s` which gathers both `stdout` and `stderr`, so no adjustment is needed. For those cases where `command | tee output` is till used I'd suggest to migrate to `rlRun -s` instead.

For those cases when test script is piping the output to another command, for example `rlRun "tmt run | grep plan`, we could add `stderr` to `stdin` using `2>&1` in order to be able to inspect both outputs from the log. What do you think?

Reply:
We decided to move to `rlRun -s` where feasiable.

Rather, introduce a dedicated test for checking correct verbosity output, as makes sense.

Reply:
I believe this is resolved.

We aim for green tests, improvments here for follow up